### ISSUE TITLE
Blank page on second request with symfony server:start

### ISSUE THREAD

**Issue by BenMorel** [symfony/symfony#32753]
**Symfony version(s) affected**:

- Symfony 4.3.2
- Symfony CLI v4.6.1

**Description**

When running `symfony server:start`, the first request goes fine, and subsequent requests return a `500` error with a blank page and no PHP error in the symfony server log:

```
Jul 26 09:54:01 |ERROR| SERVER GET  (500) /
```

To understand the problem, I added these lines to index.php:

```php
register_shutdown_function(function() {
    print_r(error_get_last());
});
```

And here is the result when the `500` error happens:

```
Array
(
    [type] => 1
    [message] => Cannot redeclare composerRequire8e9c2001e7f70743a74e6df375013049() (previously declared in /mnt/hgfs/www/project.com/vendor/composer/autoload_real.php:63)
    [file] => /mnt/hgfs/www/project.com/vendor/composer/autoload_real.php
    [line] => 63
)
```

**When using the web server bundle, it works fine.** The problem seems to lie with the Symfony CLI server only.

**How to reproduce**  

Although I can reproduce it every time on my actual project, I wasn't able to reproduce it on a fresh Symfony project so far. I can continue to try to trim down my project until the bug disappears, but am still posting this issue in the hope that it will ring a bell.

**Possible Solution**  

None so far.

**Additional context**  

I'm running Symfony in a Fedora VM running on VMware on Windows 10. The project files are located on the host and accessed by Symfony on the guest via a VMware shared folder. I first thought this could be part of the problem, until I tried the command on a copy of the project located on the Linux filesystem, and the same problem occurs.

I'm using PHP 7.3.8RC1, the server uses the PHP CGI:

```
[OK] Web server listening on http://127.0.0.1:8000 (PHP CGI 7.3.8)
```

**Comment by BenMorel** [comment]
OK, I finally managed to nail it down to the culprit: **the profiler**.

Steps to reproduce:

- `symfony new app`
- `cd app`
- `composer require --dev profiler`
- `echo 7.3 > .php-version` (might not be necessary for you)
- `symfony server:start`

Now browse the URL, it should work. Refresh, it should fail with a blank page with `500` status.

**Comment by lyrixx** [comment]
hello @BenMorel 
I tried to reproduce your issue, but I did not hit the error.
Could you try to clear the cache and retry?

**Comment by BenMorel** [comment]
@lyrixx Same thing after `bin/console cache:clear`.
Please note that the bug happens right after creating the fresh project following the steps above, so the cache should be irrelevant anyway!

**Comment by BenMorel** [comment]
Let me try on a fresh VM and I'll tell you the results.

**Comment by BenMorel** [comment]
OK, **I nailed it down to the [memprof](https://pecl.php.net/package/memprof) extension.**

When I disable it, everything works as expected.

I guess this is a bug in memprof, that only exhibits when using the profiler. Strangely, on a fresh VM, I could not reproduce it even though I installed the same PHP version and the same set of extensions: `memprof`, `opcache`, `xdebug`, `apcu`.

Looks like there is no issue in the Symfony CLI, so I'm closing this one.
